seshat-node/native/Cargo.toml

```diff
@@ -17,8 +17,8 @@ neon-build = "=0.3.3"
 [dependencies]
 neon = "=0.3.3"
 fs_extra = "1.1.0"
-serde_json = "1.0"
-r2d2 = "0.8"
-r2d2_sqlite = "0.12"
-neon-serde = "=0.2.0"
+serde_json = "1.0.44"
+r2d2 = "0.8.8"
+r2d2_sqlite = "0.13.0"
+neon-serde = "=0.3.0"
```

Comment:
ooi why are we pinning the version?

Reply:
Because the last minor version bumps of neon introduced breaking changes. And since minor bumps get automatically picked up by cargo... 

Introduced here https://github.com/matrix-org/seshat/pull/11/commits/9d1ae5cc307b76c46e892f932c43508642ff2a3d.

Reply:
Ha, :man_facepalming:  (a comment might be nice)